src/pages/DetailsPage/DetailsPage.js

```diff
@@ -49,43 +47,38 @@ export default function DetailsPage() {
               ))}
             </TagList>
           )}
-        </BodyContainer>
-      </ContentWrapper>
-    </PageContainer>
+        </ContentWrapper>
+      )}
+    </PageWrapper>
   )
 }
+const PageWrapper = styled(PageContainer)``
 
 const ContentWrapper = styled.div`
-  display: flex;
-  width: 100%;
-  flex-direction: column;
-  gap: 1rem;
-  justify-content: center;
-  align-items: center;
-`
-const HeaderContainer = styled.div`
   display: grid;
   grid-template-columns: 1fr 1fr;
-  gap: 1rem;
-  margin-top: 2rem;
-  width: min(100%, 40.7rem);
-`
-const BodyContainer = styled.div`
-  display: flex;
-  flex-direction: column;
+  grid-template-rows: repeat(4, min-content);
+  grid-template-areas:
+    'Image Header'
+    'Favorites Favorites'
+    'Description Description'
+    'Tags Tags';
```

Comment:
Seems unrelated to "React loading screen", but I like it. 
Still, it might be more complex to maintain in the future. 
Consider using the HTML element flow.

Reply:
A good point and i changed it as suggested.